### DISCUSSION TITLE
Is 7-day recall data  normalized to 1day?

### DISCUSSION THREAD

**Discussion by rbroth** [micronutrientsupport/database-architecture#114] [General]
When we get 7-day recall data, is it normalized so we only get the "average" consumption of 1 day (i.e. are all the amounts divided by 7), or do we actually get the full data? If it is normalized it would be more coinvenient because we can use the same data structure as the 24hour recall data

**Comment by LuciaSegovia** [comment]
We would get consumption over the recall period. So, if we 7-days recall, we will get the estimated food consumed over a week. I think it is best if we can have them as they are.

**Comment by rbroth** [comment]
Will the data be aggregated by week then, or will it have a day-by-day breakdown? I'm assuming the former?

Do you have some example data?

**Comment by kmtang** [comment]
I guess it might be easier to explain how the data is collected. For the household data, a surveyor will enter a household and ask one representative (usually the head female) to recall how much of each food item was consumed by the entire household in the past 7 days. Rather than going through each member of the household and trying to recall exactly what each person ate, she instead estimates on the whole how much is consumed by the household during that period. 

As an example (maybe this is relevant, maybe not...please feel free to call me out if its not), imagine a surveyor going to a home (perhaps in Germany) on any day of the week and asking the mother of that home what the family ate in the past 7 days. Rather than going through each member of the family and trying to figure out exactly what everyone ate, she would more likely try and estimate the past 7 days of food consumption of the family based off of what she cooked and what she purchased at the market. We don't know when these foods were eaten precisely, but we know that they were consumed in the past week by someone in the family. To standardize the metric, we divide the total consumption by 7 to serve as a proxy for how much was consumed per day. Clearly, the mother would most likely omit foods consumed by individuals in the household away from the home, but in rural, low-resource contexts (e.g. Malawi), families are eating out a lot less and most of the foods consumed are generally prepared at home, so this data is much more representative of the diet.

In terms of the data, we have information on the exact date that the survey was conducted or when the surveyor entered the home to ask the mother, so the household's consumption refers to the 7 days prior to this date. When I analyze this data, generally I aggregate it per month (e.g. all surveyed households conducted in June 2017) to increase my sample size since as you would expect, there is a lot of noise and error when doing recall of this nature, so larger sample sizes are generally better. 

I hope this makes sense, let me know if you need any clarification.

**Comment by rbroth** [comment]
That makes sense, thank you
